docs/plugins/subsonic.rst

```diff
@@ -0,0 +1,37 @@
+Subsonic Plugin
+================
+
+``Subsonic`` is a very simple plugin for beets that lets you automatically
```

Comment:
Use lower-case `subsonic` when you refer to the name of the plugin that the user should put into their configuration file.